src/storage/src/hummock/local_version/pinned_version.rs

```diff
@@ -126,15 +134,31 @@ impl PinnedVersion {
         self.version.id != INVALID_VERSION_ID
     }
 
-    pub fn levels(&self, new_compaction_group_id: CompactionGroupId) -> Vec<&Level> {
+    pub fn levels_by_compaction_groups_id(
+        &self,
+        compaction_group_id: CompactionGroupId,
+    ) -> Vec<&Level> {
         let mut ret = vec![];
-        if let Some(new_levels) = self.version.levels.get(&new_compaction_group_id) {
-            ret.extend(new_levels.l0.as_ref().unwrap().sub_levels.iter().rev());
-            ret.extend(new_levels.levels.iter());
+        if let Some(levels) = self.version.levels.get(&compaction_group_id) {
```

Comment:
Should we warn/panic if get results None?

Reply:
> Should we warn/panic if get results None?

ok. let's unwrap and make this a private function